tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+How to Set Up WPA Enterprise (EAP-TTLS-PAP) authentication
```

Comment:
Let's rename this file to `wpa-enterprise-eap-ttls-pap.rst` as in the future we may support other schemes and they will likely be described in different pages.